library/built-in/_prefabs/prefab_latent.ts

```diff
@@ -33,11 +33,20 @@ export const run_latent = async (p: {
 
     // case 1. start form image
     if (opts.image) {
-        const image = await p.flow.loadImageAnswer(opts.image)
-        latent = graph.VAEEncode({ pixels: image, vae: p.vae })
-        const size = graph.Image_Size_to_Number({ image: image })
+        const imageRaw = await p.flow.loadImageAnswer(opts.image)
+        // May need to use a different node for resizing
+        const image = await graph.ImageTransformResizeClip({
```

Comment:
resize the selected image, this may be a non standard node though, so might not work

Reply:
great; I think we may want to make that into an other prefab to avoid breaking too much.
also, we may want to see how practical it is to resize the image locally. it may be even simpler than to rely on any node. Cushy should provide width and height for any image that is known outside of a ComfyUI workflow run